src/Features/ShippingLabelBannerDisplayRules.php

```diff
@@ -101,6 +101,25 @@ public function __construct( $jetpack_version, $jetpack_connected, $wcs_version,
 	 * Determines whether or not the banner should be displayed.
 	 */
 	public function should_display_banner() {
+		if ( ! $this->should_allow_banner() ) {
+			return false;
+		}
+		$ab_test = get_option( 'woocommerce_shipping_prompt_ab' );
+
+		// If it doesn't exist yet, generate it for later use and save it, so we always show the same to this user.
+		if ( ! $ab_test ) {
+			$ab_test = 1 !== wp_rand( 1, 4 ) ? 'a' : 'b'; // 25% of users. b gets the prompt.
+			update_option( 'woocommerce_shipping_prompt_ab', $ab_test );
```

Comment:
```suggestion
			update_option( 'woocommerce_shipping_prompt_ab', $ab_test, false );
```
Set third param to false to disable autoload since this is an option we will not use on every page request.